spyglass/src/main/java/com/linkedin/android/spyglass/ui/MentionsEditText.java

```diff
@@ -281,16 +432,8 @@ protected MentionSpan getTouchedSpan(MotionEvent event) {
         // If one exists, call the onClick method manually
         MentionSpan[] spans = getText().getSpans(off, off, MentionSpan.class);
         if (spans.length > 0) {
-            MentionSpan span = spans[0];
-            // Do not return the span if user typed at the end of it
-            // (user may have tapped much further away in the EditText)
-            if (getText().getSpanEnd(span) == off) {
-                return null;
-            } else {
-                return span;
-            }
+            return spans[0];
```

Comment:
This change is made to avoid following scenario:
Generally for the copy/paste, selection anchors are set at the start and end of the word. Suppose MentionsEditText contains a mention and a word without any space in-between them e.g. Queen Bolton.hi where Queen Bolton is mention and .hi is plain text, the touch on last character of span i.e touch on ’n’ was considered as touch on the plain text and mention along with text was selected i.e Queen Bolton.hi is considered as a one word (basically for EditText Bolton.hi is a one word but as because Queen Bolton is a mention, selection anchor is adjusted at the start of the span.).
This case is avoided by considering touch on last character of span as touch on span and as per our implementation if span is long clicked whole span is selected for copy paste action.